src/models/signUp.enum.ts

```diff
@@ -1,8 +1,8 @@
 export enum SignUpFields {
-  firstName = 'firstName',
-  lastName = 'lastName',
-  email = 'email',
-  password = 'password',
-  confirmPassword = 'confirmPassword',
-  phoneNumber = 'phoneNumber'
+  FIRSTNAME = 'firstName',
+  LASTNAME = 'lastName',
+  EMAIL = 'email',
+  PASSWORD = 'password',
+  CONFIRMPASSWORD = 'confirmPassword',
+  PHONENUMBER = 'phoneNumber'
```

Comment:
I meant capitalize, not uppercase. But is up to you.

Reply:
Sorry. I`d like to use capitalize for now if it is not a problem.

Reply:
Of course, it is up to you. Just don`t use the lower case.